Extensions/UserDtoExtensions.cs

```diff
@@ -12,8 +12,8 @@ public static AppUser ToAppUser(this UserDto userDto)
             {
                 UserName = userDto.UserName,
                 Email = userDto.Email,
-                Links = userDto.Links,
-                Image = userDto.Image
+                //Links = userDto.Links,
```

Comment:
Is there any reason for removing links from Appuser entity?

Reply:
For now, I have commented it just to pass the data from UI to backend and complete the registration process. Also, I think we have to change the datatype of Links from List<string> to List<Dictionary<string,string>> because of the composition of its data. What do you think?

Reply:
Probably, but just Dictionary<string,string> to store it as name-link pairs

Reply:
Noted on this. But I have no idea yet on how to entry the dictionary data to the Links column as JSON as I haven't interacted with this kind of data before.